apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -990,7 +988,7 @@ do_bpapi_call(Node, Call, Args) ->
 do_bpapi_call_vsn(SupportedVersion, Call, Args) ->
     case lists:member(SupportedVersion, supported_versions(Call)) of
         true ->
-            apply(emqx_bridge_proto_v3, Call, Args);
+            apply(emqx_bridge_proto_v4, Call, Args);
```

Comment:
Nit: isn't this a kind of evading from some of the bpapi checks? 🤔

Reply:
What scenario do you have in mind? I believe that piece of logic always (silently) assumed that the type (i.e. signature) of an existing RPC does not change across protocol versions. Which is general is not true, but I'm not aware of bpapi checks for that.

Reply:
I mean, dialyzer won't check that `Args` type matches the bpapi's wrapper type; only that bpapi's wrapper declared type matches the type of the corresponding remote call.